sdk/src/Services/DynamoDBv2/Custom/DataModel/Configs.cs

```diff
@@ -130,6 +131,11 @@ public DynamoDBContextConfig()
         /// otherwise exceptions may be thrown and/or the results of certain DynamoDB operations may change.
         /// </remarks>
         public bool? DisableFetchingTableMetadata { get; set; }
+
+        /// <summary>
+        /// If true, all <see cref="DateTime"/> properties are retrieved in UTC timezone while reading data from DynamoDB. Else, the local timezone is used.
```

Comment:
Can you make this clear this is for the high level library. The service client itself is going to always use UTC.